docs/user/tensorflow.md

```diff
@@ -0,0 +1,157 @@
+    * Make sure the environment variable **LIBRARY_PATH** includes the paths to all daos libraries
+    * Make sure the environment variable **LD_LIBRARY_PATH** includes the paths to:
+        * All daos libraries
+        * The tensorflow framework (libtensorflow and libtensorflow_framework)
+    * If not, find the required libraries and add their paths to the environment variable
+      ```
+      export LD_LIBRARY_PATH="<path-to-library>:$LD_LIBARY_PATH"
+      ```
+    * Make sure the environment variable **CPLUS_INCLUDE_PATH** and **C_INCLUDE_PATH** includes the paths to:
+        * The tensorflow headers (usually in /usr/local/lib64/python3.6/site-packages/tensorflow/include)
+    * If not, find the required headers and add their paths to the environment variable
+      ```
+      export CPLUS_INCLUDE_PATH="<path-to-headers>:$CPLUS_INCLUDE_PATH"
+      export C_INCLUDE_PATH=$CPLUS_INCLUDE_PATH:$C_INCLUDE_PATH
+      ```
+
+      * Build the project using bazel
+        ```
+        bazel build --action_env=LIBRARY_PATH=$LIBRARY_PATH -s --verbose_failures //tensorflow_io/... //tensorflow_io_gcs_filesystem/...
+        ```
+        This should take a few minutes. 
+
+        Note that sandboxing may result in build failures when using 
+        Docker Containers for DAOS due to mounting issues, if that’s the case, 
+        add **--spawn_strategy=standalone** to the above build command to 
+        bypass sandboxing. (When disabling sandbox, an error may be thrown for 
+        an undefined type z_crc_t due to a conflict in header files. 
+        In that case, find the crypt.h file in the bazel cache in subdirectory 
+        /external/zlib/contrib/minizip/crypt.h and add the following line to the
+        file **typedef unsigned long z_crc_t;** then re-build).
+
+
+
+### Testing
+Assuming you are in a terminal in the repository root directory:
+
+* Run the following command for the simple serial test to validate building. Note that any tests need to be run with the TFIO_DATAPATH flag to specify the location of the binaries.
+  ```
+  $ TFIO_DATAPATH=bazel-bin python3 -m pytest -s -v tests/test_serialization.py
+
+  ```
+
+* Run the following commands to run the dfs plugin test:
+  ```
+  # To create the required pool and container and export required env variables for the dfs tests.
+  $ source tests/test_dfs/dfs_init.sh
+  # To run dfs tests
+  $ TFIO_DATAPATH=bazel-bin python3 -m pytest -s -v tests/test_dfs.py
+  # For Cleanup, deletes pools and containers created for test.
+  $ bash ./tests/test_dfs/dfs_cleanup.sh
+  ```
+
+## User Guide
+
+To use the Tensorflow-IO Library, you'll need to import the required packages 
+as follows:
+
+```python
+import tensorflow as tf
+import tensorflow_io as tfio
+```
+
+To use the DFS Plugin, all that needs to be done is to supply the paths of the required
+files/directories in the form of a DFS URI:
+
+dfs://<Pool-Label-or-UUID>/<Cont-Label-Or-UUID>/<Path>
```

Comment:
would this work if one has a container with a UNS path?
in the implementation do you use duns_resolve_path() ?

Reply:
Yes, the plugin supports UNS paths

Reply:
so you could update this to include support for using a path also?
so in addition to dfs://pool/cont/path
you can add
dfs://path where path includes the path to the DAOS container
let's say if you have create a container with --path=/mnt/dfuse/cont1
would passing dfs:/mnt/dfuse/cont1/path work?

Reply:
@omaraziz255 do you intend to update the patch based on Mohamad's comment?

Reply:
Yes, sorry for the late response. I'll update it asap